packages/remark-lint-blockquote-indentation/index.js

```diff
@@ -42,15 +42,15 @@
  * @copyright 2015 Titus Wormer
  * @license MIT
  * @example
- *   {"name": "ok.md", "setting": 4}
+ *   {"name": "ok.md", "config": 4}
```

Comment:
I noticed that we currently use `config` in all the readmes: https://github.com/remarkjs/remark-lint/blob/014fca79e0be563ca9a65044b7a93ca165bd3a9a/script/build-rules.js#L540
What do you think about replacing `setting` with that?

Reply:
yeah sure!

Reply:
Thanks!